website/blog/2018-02-23-1.11.0.md

```diff
@@ -189,7 +189,7 @@ const abc = props.something
 
 ### CSS/SCSS/Less
 
-#### Lists of lists now have prettier formatting ([#3930] by [@evilebottnawi])
+#### Lists of lists now have Prettier formatting ([#3930] by [@evilebottnawi])
```

Comment:
This should still be `prettier`.

Reply:
Sure.. but why?

Reply:
The PR improved the formatting of lists of lists, but we already formatted them.

Reply:
and that's why it should be written `prettier` instead of `Prettier` or am I missing something?

Reply:
Yes that’s the reason.

Reply:
😕

Reply:
I guess it’s because prettier = better here, it’s not a library name in this context.

Reply:
might even be worth using a different adjective to avoid confusion:

"Lists of lists now have cleaner formatting"
"Lists of lists now have better formatting"

or something similar

Reply:
Maybe ‘more consistent’ or just ‘consistent’? ‘Better’ is way too subjective IMO, yet ‘cleaner’ looks OK.

Agree about the confusion with `prettier’ here.

Reply:
Now I realised why I was confused.. :)